src/main/java/com/mraof/minestuck/block/MSBlocks.java

```diff
@@ -274,7 +275,7 @@ public static void registerBlocks(final RegistryEvent.Register<Block> event)
 		registry.register(new Block(Block.Properties.create(Material.ROCK).hardnessAndResistance(1.5F, 6.0F).harvestTool(ToolType.PICKAXE).harvestLevel(0)).setRegistryName("flowery_mossy_cobblestone"));
 		registry.register(new Block(Block.Properties.create(Material.ROCK).hardnessAndResistance(1.5F, 6.0F).harvestTool(ToolType.PICKAXE).harvestLevel(0)).setRegistryName("flowery_mossy_stone_bricks"));
 		registry.register(new Block(Block.Properties.create(Material.ROCK, MaterialColor.SAND).hardnessAndResistance(3.0F, 9.0F).harvestTool(ToolType.PICKAXE).harvestLevel(0)).setRegistryName("coarse_end_stone"));
-		registry.register(new EndGrassBlock(Block.Properties.create(Material.ROCK, MaterialColor.PURPLE).hardnessAndResistance(3.0F, 9.0F).harvestTool(ToolType.SHOVEL).tickRandomly()).setRegistryName("end_grass"));
+		registry.register(new EndGrassBlock(Block.Properties.create(Material.ROCK, MaterialColor.PURPLE).hardnessAndResistance(3.0F, 9.0F).harvestTool(ToolType.SHOVEL).tickRandomly()).setRegistryName("end_grass_block"));
```

Comment:
Since you're changing a registry name here, and I'd say that we want to keep some level of backwards-compability at this time, I'd like you to map the old name back to the block during the MissingMappings event, similarly to what we're already doing in MSBiomes.

Reply:
I will try and do this!

Reply:
The MissingMappings-backwards compatibility does not appear to be working, so a name revert may be the best solution for the moment